packages/cli/src/utils.ts

```diff
@@ -1,10 +1,13 @@
-import { ContentType, MedplumClient } from '@medplum/core';
+import { ContentType, MedplumClient, encodeBase64 } from '@medplum/core';
 import { Bot, Extension, OperationOutcome } from '@medplum/fhirtypes';
 import { existsSync, readFileSync, writeFile } from 'fs';
-import { basename, extname, resolve } from 'path';
+import { basename, extname, resolve, join } from 'path';
 import internal from 'stream';
 import tar from 'tar';
 import { FileSystemStorage } from './storage';
+import { SignJWT } from 'jose';
+import { createPrivateKey, randomBytes, createHmac } from 'crypto';
+import { homedir } from 'os';
```

Comment:
Please run "Organize imports" in VS Code

<img width="460" alt="image" src="https://github.com/medplum/medplum/assets/749094/aaf8af46-b2e1-4324-bce7-eac372ced034">

(Note... we should make this a linting rule somehow)